docs/reference/commandline/trust_signer_add.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "signer-add"
+description: "The signer-remove command description and usage"
+keywords: "signer, notary, trust"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# trust signer-add
+
+```markdown
+Usage:  docker trust signer-add [OPTIONS] signer IMAGE [IMAGE...]
+
+Add a signer to an image
```

Comment:
Unrelated side note: Perhaps we should reword this description when defining the subcommand as this can be used to add a signer to one or more images.